processes/host-selection/host-prioritization.md

```diff
@@ -0,0 +1,37 @@
+# Meeting to decide host prioritization
+
+**DATE:** 2019-23-01
+
+**PARTICIPANTS:** @Jethet, @eskape16, @daverick, @amaliacardenas
+**LOCATION:** Cloud Co-working
+
+## TL;DR
+We want to be more transparent with the way we organize events. In this way anyone who attends or coaches at codebar knows our needs and if interested can help us out. The reason why we want to do this is to have more members from the community participate in the organization of events and to help us as we expand.
+
+In this document, you will find
+1. The criteria that we use when we prioritize where to host codebar events.
+2. A guide for how to contact hosts who want to sponsor a workshop.
+
+# Host prioritization criteria
+
+## Must Have
+* **Aligns with the codebar mission**
+* **Safe place**
+* **Food and drinks**
+* **Comfortable space for workshops**
+
+## Important
+* **Accessibility within the city**
+* **Maximum capacity**
+
+## Nice to have
+* **Comfortable space for talks**
+* **Sustainable company**
```

Comment:
To be honest, this is probably more a personal priority and maybe should not come up at all. Although I would  have great difficulty to go to a workshop hosted by an arms dealer or a company such as Monsanto, it is not something that is part of the Codebar code of conduct or anything like that. The mission of Codebar is very much focused on a safe environment, fighting sexism etc. and it would be a mistake to start championing other issues, however important they may be. It probably goes beyond what we can ask or do to interfere with the plastic cups and throw-away water bottles that our hosts offer.
Suggestion: we can keep this in the back of our minds when considering hosts but it cannot be a general requirement that we ask our hosts to comply with.